charts/snapshot-warmer/scripts/snapshot-warmer.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/sh
+
+cd /
+
+## Snapshot Namespace
+NAMESPACE="${NAMESPACE}" yq e -i '.metadata.namespace=strenv(NAMESPACE)' createVolumeSnapshot.yaml
```

Comment:
I don't think we should need to prefix with `NAMESPACE="${NAMESPACE}"`. It is assigning a variable's value to the value it already is. It is a global env var passed in to the container via the pod specification and should be available everywhere. This comment applies to multiple places in each of the snapshotting bash scripts we have.
```suggestion
yq e -i '.metadata.namespace=strenv(NAMESPACE)' createVolumeSnapshot.yaml
```

Reply:
This is yq specific syntax. Check it out -

```bash
$ NAMESPACE=test
$ yq e -i '.metadata.namespace=strenv(NAMESPACE)' createVolumeSnapshot.yaml
$ cat createVolumeSnapshot.yaml
metadata:
  namespace: ""
$ NAMESPACE="${NAMESPACE}" yq e -i '.metadata.namespace=strenv(NAMESPACE)' createVolumeSnapshot.yaml
$ cat createVolumeSnapshot.yaml
metadata:
  namespace: "test"
```

Reply:
I just tested it with using `export NAMESPACE` without prepending the env var to the `yq` cmd and it works. Export makes it a global var. The env vars are being passed into the container from the pod as global env vars, so this should work. Was this tested before and it didn't work?
```shell
❯  NAMESPACE=test
❯  yq e -i '.metadata.namespace=strenv(NAMESPACE)' createVolumeSnapshot.yaml
❯  cat createVolumeSnapshot.yaml | head -5
apiVersion: snapshot.storage.k8s.io/v1
kind: VolumeSnapshot
metadata:
  name: "2021-11-07-01-13-02-node-snapshot"
  namespace: ""
❯ export NAMESPACE=test
❯ yq e -i '.metadata.namespace=strenv(NAMESPACE)' createVolumeSnapshot.yaml
❯ cat createVolumeSnapshot.yaml | head -5
apiVersion: snapshot.storage.k8s.io/v1
kind: VolumeSnapshot
metadata:
  name: "2021-11-07-01-13-02-node-snapshot"
  namespace: "test"